scripts/create-instance.sh

```diff
@@ -108,13 +108,8 @@ set -o nounset
 
 if [[ "$@" != *"--devel"* ]]; then
 # sphinxdoc-install-invenio-full-begin
-# FIXME the next pip commands are needed only for invenio<3.0.0a3
-pip install invenio-db[postgresql] --pre
```

Comment:
Just testing the updated branch, and got into:

```console
$ docker-compose run --rm web ./scripts/populate-instance.sh
[...]
  File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/flask_sqlalchemy/__init__.py", line 542, in get_engine
    self._sa.apply_driver_hacks(self._app, info, options)
  File "/home/invenio/.virtualenvs/invenio3/lib/python2.7/site-packages/invenio_db/shared.py", line 63, in apply_driver_hacks
    from psycopg2.extensions import adapt, register_adapter
ImportError: No module named psycopg2.extensions
```

Looks like the explicit `pip install invenio-db[postgresql] --pre` instruction steps may still be needed?